Prueba1.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Support.UI;
+
+namespace SeleniumTests
```

Comment:
I'm not sure but I think that the name of the `namespaces` should be in the singular shape: `SeleniumTest`. A better practice could be using a more meaningful name like: `ProjectNameTestSuitcase`.